packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -172,6 +172,8 @@ export class SDKClient {
       ),
     );
 
+    ethereumTransaction.setMaxGasAllowanceHbar(ConfigService.get('MAX_GAS_ALLOWANCE_HBAR'));
```

Comment:
I'd like to include a comment to the docs for reference, _i.e._, https://docs.hedera.com/hedera/sdks-and-apis/sdks/smart-contracts/ethereum-transaction